src/context/AppProvider/AppContext.tsx

```diff
@@ -170,7 +170,7 @@ export const AppProvider = ({ children }: { children: React.ReactNode }) => {
                 adapter as bitcoin.ChainAdapter
               ).getSupportedAccountTypes()
               for (const accountType of supportedAccountTypes) {
-                const accountParams = utxoAccountParams(bitcoin, accountType, 0)
+                const accountParams = utxoAccountParams(accountType, 0)
```

Comment:
we can remove the whole bitcoin shit check above now too

Reply:
Addressed in https://github.com/shapeshift/web/pull/1908/commits/7cd8a1b29941e6cf011fe4054a4052058314b857